app/assets/javascripts/osem-schedule.js

```diff
@@ -86,8 +133,18 @@ $(document).ready( function() {
     snap: '.schedule-room-slot',
     revertDuration: 200,
     revert: function (event, ui) {
-        console.log(event.attr);
-        return !event;
+      console.log(event.attr);
```

Comment:
undefined
This raises an error. Doesn't affect the PR's functionality, which seems to be working OK.

```
osem-schedule.js?body=1:136 function ( name, value ) {
		return access( this, jQuery.attr, name, value, arguments.length > 1 );
	}

```